### PR TITLE
Hotfix/unify abandoned cart fields

### PR DESCRIPTION
**Pull request by sinukaarel**
Unify abandoned cart fields

- firstname -> first_name
- lastname -> last_name
- product_description_short -> product_description

Added
- product_base_price

Removed
- store_url

### CODE REVIEW

**Comment by sinukaarel** [comment]
No `product_sku` currently and `over_10_products`. Should I add those?

File: controllers/front/SmailyCartCron.php [hunk 1]
```diff
@@ -45,107 +45,157 @@ public function init()
...
+            if (in_array('last_name', $sync_fields)) {
+                $adresses['last_name'] = $abandoned_cart['lastname'];
+            }
+
+            // Populate abandoned cart with empty values for legacy api.
+            $fields_available = array(
+                'name',
+                'description',
+                'price',
+                'category',
```
Comment (sinukaarel) [inline controllers/front/SmailyCartCron.php]: The product category is not a field that is used by other plugins. Should I remove it from Prestashop also?

File: controllers/front/SmailyCartCron.php [hunk 2]
```diff
@@ -45,107 +45,157 @@ public function init()
...
+                        case 'base_price':
+                            $adresses['product_base_price_' . $count] = Tools::displayPrice(
+                                $product['price_without_reduction']
+                            );
+                            break;
+                        case 'price':
+                            $adresses['product_price_' . $count] = Tools::displayPrice(
+                                $product['price_with_reduction']
+                            );
+                            break;
```
Comment (sinukaarel) [inline controllers/front/SmailyCartCron.php]: `base_price` and `price` fields now have also formating and pass values as displayed in the store. Same as Magento 2 plugin

File: controllers/front/SmailyCartCron.php (hunk 1 above)
Comment (kaittodesk) [reply controllers/front/SmailyCartCron.php]: If it is not used by other plugins, then I suggest we remove it.

File: controllers/front/SmailyCartCron.php (hunk 2 above)
Comment (kaittodesk) [reply controllers/front/SmailyCartCron.php]: Awesome!

File: controllers/front/SmailyCartCron.php (hunk 1 above)
Comment (sinukaarel) [reply controllers/front/SmailyCartCron.php]: Removed in https://github.com/sendsmaily/smaily-prestashop-module/pull/12/commits/2a56e8af7442151d91968af40dbc202f4bc9a7ae

**Pull request merged**